readthedocs/projects/tasks.py

```diff
@@ -845,9 +845,11 @@ def build_docs_html(self):
         if self.build_force:
             html_builder.force()
         html_builder.append_conf()
+        html_builder.clean()
```

Comment:
We should refactor `build_docs_html` to use `build_docs_class`, but not so trivial code needs to be moved around, so keeping simple this for now. I'm creating a new issue to refactor this later.

Reply:
https://github.com/rtfd/readthedocs.org/issues/5410